src/postcapture.rs

```diff
@@ -118,7 +114,42 @@ impl PostCaptureAction for CopyToClipboard {
     }
 }
 
+/// Executes the post capture actions in the order they are defined in the settings.
+pub fn do_postcapture_actions(history_model: &HistoryModel, conn: &SqliteConnection, pixbuf: &mut Pixbuf) {
+    for action in get_actions_from_settings() {
+        action.handle(history_model, conn, pixbuf)
+    }
+}
+
+fn get_actions_from_settings() -> Vec<&'static dyn PostCaptureAction> {
+    let settings = gio::Settings::new("kc.kcshot");
+    let actions: Vec<String> = settings.strv("post-capture-actions")
+        .iter()
+        .map(|gstr| gstr.to_string())
+        .collect();
+
+    let action_lookup: HashMap<String, &dyn PostCaptureAction> = get_postcapture_actions()
+        .iter()
+        .map(|action| (action.id(), *action))
+        .collect();
+
+
+    let mut actions_todo = Vec::new();
```

Comment:
If we rename `do_postcapture_actions` to `run_postcapture_actions` as I proposed in an earlier comment, this should be renamed to `actions_to_run`, if we do not, this should be renamed to `actions_to_do` (I parse `todo` as a single word)